internal/util/paramtable/component_param.go

```diff
@@ -915,6 +917,10 @@ func (p *dataCoordConfig) initSegAssignmentExpiration() {
 	p.SegAssignmentExpiration = p.Base.ParseInt64WithDefault("dataCoord.segment.assignmentExpiration", 2000)
 }
 
+func (p *dataCoordConfig) initSegmentMaxLifetime() {
+	p.SegmentMaxLifetime = time.Duration(p.Base.ParseInt64WithDefault("dataCoord.segment.maxLife", 24*60*60)) * time.Second
```

Comment:
put this dataCoord.segment.maxLife to milvus.yaml

Reply:
done